src/auto-evo/RunResults.cs

```diff
@@ -169,17 +169,24 @@ string PatchString(Patch patch)
 
             void OutputPopulationForPatch(Species species, Patch patch, long population)
             {
-                builder.Append("  ");
-
-                builder.Append(PatchString(patch));
-
-                builder.Append(" population: ");
-                builder.Append(Math.Max(population, 0));
+                if (population > 0)
+                {
+                    builder.Append("  ");
+                    builder.Append(PatchString(patch));
+                    builder.Append(" population: ");
+                    builder.Append(Math.Max(population, 0));
 
-                if (previousPopulations != null)
+                    if (previousPopulations != null)
+                    {
+                        builder.Append(" previous: ");
+                        builder.Append(previousPopulations.GetPatch(patch.ID).GetSpeciesPopulation(species));
+                    }
```

Comment:
I think the previous population should be shown even when going extinct.

Reply:
Why do you enjoy making my life painful?

Reply:
I want features to be done right the first time, instead of needing a follow up PR to fix things up (which might not happen in the worst case, so I try to only accept PRs that don't need future fixing that might not happen).